iree_tests/conftest.py

```diff
@@ -375,12 +375,19 @@ def runtest(self):
             raise e
 
     def test_compile(self):
-        proc = subprocess.run(self.compile_args, capture_output=True, cwd=self.test_cwd)
+        compile_env = os.environ.copy()
+        compile_env["IREE_TEST_PATH_EXTENSION"] = os.getenv("IREE_TEST_PATH_EXTENSION", default=self.test_cwd)
+        path_extension = compile_env["IREE_TEST_PATH_EXTENSION"]
+        cmd = subprocess.list2cmdline(self.compile_args)
+        cmd = cmd.replace("${IREE_TEST_PATH_EXTENSION}", f"{path_extension}")
+        proc = subprocess.run(cmd, env=compile_env, shell=True, capture_output=True, cwd=self.test_cwd)
```

Comment:
`IREE_TEST_PATH_EXTENSION` could be documented in the README at https://github.com/nod-ai/SHARK-TestSuite/tree/main/iree_tests and this code could have a comment, but that can be done later as we continue to refactor and make use of this

Reply:
sounds good